huaweicloud/services/acceptance/identitycenter/resource_huaweicloud_identitycenter_user_test.go

```diff
@@ -0,0 +1,146 @@
+package identitycenter
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getIdentityCenterUserResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+	// getIdentityCenterUser: Query Identity Center user
+	var (
+		getIdentityCenterUserHttpUrl = "v1/identity-stores/{identity_store_id}/users/{user_id}"
+		getIdentityCenterUserProduct = "identitystore"
+	)
+	getIdentityCenterUserClient, err := cfg.NewServiceClient(getIdentityCenterUserProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating IdentityCenter Client: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error creating Identity Center client: %s", err)
```

please modify others

Reply:
done